File: Basics/For Loop/Exercise/05.py
count_tab = int(input())
salary = int(input())

left_money = salary
for i in range(1, count_tab + 1):
    web_page = input()
    if web_page == "Facebook":
        left_money -= 150
    elif web_page == "Instagram":
        left_money -= 100
    elif web_page == "Reddit":
        left_money -= 50
    else:
        continue

    if left_money == 0:
        print("You have lost your salary.")
        break

if left_money > 0:
    print(f"{left_money:.0f}")

File: JS-Basics/For Loop/Exercise/05.js
function calculateRemainingSalary(input) {
    const n = input[0];
    let salary = input[1]; 

    const penalties = {
        "Facebook": 150,
        "Instagram": 100,
        "Reddit": 50
    };

    for (let i = 2; i < input.length; i++) {
        const site = input[i];
        if (penalties[site]) {
            salary -= penalties[site];
        }
        if (salary <= 0) {
            console.log("You have lost your salary.");
            return;
        }
    }

    console.log(salary);
}
